tests/UpdatePaymentMethod/UpdatePaymentMethodBuilderTest.php

```diff
@@ -0,0 +1,175 @@
+<?php
+
+
+namespace Laravel\Cashier\Tests\UpdatePaymentMethod;
+
+use Illuminate\Support\Facades\Event;
+use Laravel\Cashier\Events\MandateUpdated;
+use Laravel\Cashier\FirstPayment\Actions\AddBalance;
+use Laravel\Cashier\FirstPayment\FirstPaymentHandler;
+use Laravel\Cashier\Order\Order;
+use Laravel\Cashier\Tests\BaseTestCase;
+use Laravel\Cashier\Tests\Fixtures\User;
+use Mollie\Api\MollieApiClient;
+use Mollie\Api\Resources\Payment;
+
+class UpdatePaymentMethodBuilderTest extends BaseTestCase
```

Comment:
The class name tells me that it is testing the `UpdatePaymentMethodBuilder`, but it seems not to be the case.

Reply:
So it's important here to clear up what is being tested exactly.
Also, are there "non-happy paths" we need to test for?

Reply:
Class name renamed

Reply:
>The class name tells me that it is testing the UpdatePaymentMethodBuilder, but it seems not to be the case.

Class renamed

> So it's important here to clear up what is being tested exactly.
> Also, are there "non-happy paths" we need to test for?

I don't think so